### ISSUE TITLE
44 Draco_meteor-Home security and Automation

### ISSUE THREAD

**Issue by codetronaut** [CodeBreakMIT/CodeBreak1.0#14]
**Before you start, please follow this format for your issue title**:  
44 Draco_meteor-Home security and automation

## ℹ️ Project information
- **Project Name**: Home security and automation
- **Short Project Description**: This project aims to maintain the security of your home from anywhere without worrying about it.
- **Team ID**: 44
- **Team Name**: Draco_meteor
- **Team Members**: Rajendra Prasad : @RajendraPrasad686 and Anmol: @codetronaut
- **Demo Link**: _Upload your project demo on YouTube/Drive and paste its link here_
- **Repository Link**: https://github.com/codetronaut/home-sec-auto
- **Labels**: Machine Learning

## 🔥 Your Pitch
The objective of this project is to build the machine learning model which detect the unknown person on the gate of 
Home.
The project work as follow:
The image of the person will be capture through the camera and and check whether person is known or unknown.
If the person is known then gate will be opened automatically else it will depend on the owner for permission. Face detection is done through using Convolutional Neural Network(CNN) recognition algorithm as a classifier.

## Technology Stack Used
Python, Machine Learning, MongoDB, OpenCV and Tensorflow.

## 🔦 Any other specific thing you want to highlight?
_(Optional)_

## ✅ Checklist

**Before you post the issue**:
- [x] You have followed the issue title format.
- [x] You have mentioned the correct labels.
- [x] You have provided all the information correctly.
